resources/platforms.json

```diff
@@ -4904,6 +4955,7 @@
         "Platform": "JAVA",
         "Platform_Maker": "Oracle",
         "Platform_Version": "unknown",
+        "Platform_Description": "unknown",
```

Comment:
I noticed a case where I set the platform to "JAVA" for a pattern that was inside a useragent group that was set to "WinMobile" as the platform, and while I did get "JAVA" as the "platform" for the test, the "platform_description" was still "Windows Mobile OS". 

I've run into this before where the properties of the group's platform (if defined) will be used if the overridden platform doesn't contain those properties.  I set this to "unknown" as I felt it would have less impact (as it should be "unknown" already in the case that it's not overriding another platform).